modules/util/portal-tools-soy-builder/src/test/resources/com/liferay/portal/tools/soy/builder/commands/dependencies/build_soy/expected/text_localizable.soy.js

```diff
@@ -22,7 +22,7 @@ soy.$$registerDelegateFn(soy.$$getDelTemplateId('ddm.field'), 'text_localizable'
+  output += '<div class="form-group' + soy.$$escapeHtmlAttribute(opt_data.visible ? '' : ' hide') + ' liferay-ddm-form-field-text-localizable ' + soy.$$escapeHtmlAttribute(opt_data.tip ? 'liferay-ddm-form-field-has-tip' : '') + '" data-fieldname="' + soy.$$escapeHtmlAttribute(opt_data.name) + '"><label class="control-label" for="' + soy.$$escapeHtmlAttribute(opt_data.name) + '">' + soy.$$escapeHtml(opt_data.label) + ((opt_data.required) ? '<span class="icon-asterisk text-warning"></span>' : '') + '</label>' + ((opt_data.tip) ? '<p class="liferay-ddm-form-field-tip">' + soy.$$escapeHtml(opt_data.tip) + '</p>' : '') + '<div class="input-group-container ' + ((opt_data.tooltip) ? 'input-group-default' : '') + ' input-localized">' + ((opt_data.displayStyle == 'multiline') ? '<textarea aria-describedby="' + soy.$$escapeHtmlAttribute(opt_data.name) + '_desc" class="field form-control language-value" dir="' + soy.$$escapeHtmlAttribute(opt_data.dir) + '" id="' + soy.$$escapeHtmlAttribute(opt_data.name) + '" name="' + soy.$$escapeHtmlAttribute(opt_data.name) + '" placeholder="' + soy.$$escapeHtmlAttribute(opt_data.placeholder) + '" ' + ((opt_data.readOnly) ? 'readonly' : '') + ' >' + soy.$$escapeHtmlRcdata(displayValue__soy5) + '</textarea>' : '<input aria-describedby="' + soy.$$escapeHtmlAttribute(opt_data.name) + '_desc" class="field form-control language-value" dir="' + soy.$$escapeHtmlAttribute(opt_data.dir) + '" id="' + soy.$$escapeHtmlAttribute(opt_data.name) + '" name="' + soy.$$escapeHtmlAttribute(opt_data.name) + '" placeholder="' + soy.$$escapeHtmlAttribute(opt_data.placeholder) + '" ' + ((opt_data.readOnly) ? 'readonly' : '') + ' type="text" value="' + soy.$$escapeHtmlAttribute(displayValue__soy5) + '">') + '<div class="input-localized-content hidden" id="' + soy.$$escapeHtmlAttribute(opt_data.name) + 'ContentBox" role="menu">' + ((opt_data.availableLocalesMetadata) ? ddm.available_locales(opt_data) : '') + '</div>' + ((opt_data.tooltip) ? '<span class="input-group-addon"><span class="input-group-addon-content"><a class="help-icon help-icon-default icon-monospaced icon-question" data-original-title="' + soy.$$escapeHtmlAttribute(opt_data.tooltip) + '" href="javascript:void(0);" title="' + soy.$$escapeHtmlAttribute(opt_data.tooltip) + '"></a></span></span>' : '') + '</div>' + ((opt_data.childElementsHTML) ? soy.$$filterNoAutoescape(opt_data.childElementsHTML) : '') + '</div>';
```

Comment:
Hey @georgel-pop-lr, you touched a generated file here. If you caught any CI errors, I think you may take a look at this

Reply:
Hi @diegonvs,

Thanks for looking, I see now he stable tests passed.

As there are many files affected, could we ask some help from someone on QA? 

Regards.

Reply:
> Thanks for looking, I see now the stable tests passed.

<img src="https://media1.giphy.com/media/sy843WuhjhB5K/giphy.gif"/>

> As there are many files affected, could we ask some help from someone on QA?

Yes, @john-co, taking a look on LocalFile.FrontendDataSet#ActionsCanBeCustomized, It looks like we haven´t modified anything there. Same for LocalFile.FormsColorField#PredefinedValueCanBeAdded

Reply:
Same for [LocalFile.OpenIDConnect#OpenIDConnectSmokeWithAuth0](https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/32207//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_OpenIDConnect_OpenIDConnectSmokeWithAuth0_)